src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -285,6 +291,18 @@
       text: az spring-cloud app identity show -n MyApp -s MyCluster -g MyResourceGroup
 """
 
+helps['spring-cloud app identity force-set'] = """
+    type: command
+    short-summary: Force set managed identities on an app.
+    examples:
+    - name: Force remove all managed identities on an app.
+      text: az spring-cloud app identity force-set -n MyApp -s MyCluster -g MyResourceGroup --system-assigned disable --user-assigned disable
+    - name: Force remove all user-assigned managed identities on an app, and enable or keep system-assigned managed identity.
+      text: az spring-cloud app identity force-set -n MyApp -s MyCluster -g MyResourceGroup --system-assigned enable --user-assigned disable
+    - name: Force remove system-assigned managed identity on an app, and assign or keep user-assigned managed identities.
+      text: az spring-cloud app identity force-set -n MyApp -s MyCluster -g MyResourceGroup --system-assigned disable --user-assigned IdentityResourceId1 IdentityResourceId2

```

Comment:
You want to implement the command to remove managed  identity, right?
If so, please refer [operate-managed-identity-on-existing-resource](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md#operate-managed-identity-on-existing-resource) to use `az <resource> identity remove` 
> 2. Remove identities with identity remove command